lmvachon/bad.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+
+<head>
+	<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+	<title>Lauren's Bad Code Example</title>
+
+	<link rel="stylesheet" href="/css/style.css" type="text/css" media="all" />
+</head>
+
+<body>
+
+	<header>
+		<nav>
+			<a href="#">Home
+			<a href="#">About
+			<a href="#">Contact
+		</nav>
```

Comment:
Nice use of the HTML5 `<nav>` tag, much more semantic than a `<div>`. The way we style navigation blocks, we usually wrap them up in unordered lists, with each link as its own `<li>` for ease of styling. Additionally, you'll want to close these '<a>' tags so the links work as expected. A tool I like to use to help with HTML page structure and making sure my elements all have closing tags is https://emmet.io/. Worth looking into if it's compatible with your favorite code editor.